### PR TITLE
Add external_svc_subnet for k8s loadbalancer type service

### PR DESCRIPTION
**Pull request by jihoon5**
Add external_svc_subnet field for k8s loadbalancer type service
in kuryr-kubernetes code support external service subnet
but openshift does not support yet

### CODE REVIEW

**Comment by michaelgugino** [comment]
@dulek  @luis5tb Any thoughts?

/ok-to-test

File: roles/kuryr/templates/configmap.yaml.j2 [hunk 1]
```diff
@@ -149,6 +149,9 @@ data:
     # The driver to determine Neutron subnets for pod ports (string value)
     pod_subnets_driver = default
 
+    # The driver to determine Neutron subnets for LoadBalancers (string value)
+    external_svc_subnet = default
```
Comment (dulek) [inline roles/kuryr/templates/configmap.yaml.j2]: This option shouldn't be in [kubernetes] section, but [neutron_defaults] section. Also the value does not make sense. It's supposed to be id of the subnet.

File: roles/kuryr/templates/configmap.yaml.j2 [hunk 2]
```diff
@@ -229,6 +232,7 @@ data:
     # TODO (apuimedo): Remove the duplicated line just after this one once the
     # RDO packaging contains the upstream patch
     worker_nodes_subnet = {{ kuryr_openstack_worker_nodes_subnet_id }}
+    external_svc_subnet = {{ kuryr_openstack_external_svc_subnet_id }}
```
Comment (dulek) [inline roles/kuryr/templates/configmap.yaml.j2]: This one is okay.

**Comment by jihoon5** [comment]
PTAL!

**Comment by jihoon5** [comment]
I'm Sorry, 
How can I merge this code?

**Comment by michaelgugino** [comment]
/lgtm

Code typically merges after it has a /lgtm from a project maintainer.

**Pull request merged**